mypy.ini

```diff
@@ -0,0 +1,14 @@
+[mypy]
+python_version = 3.10
+exclude = examples
+ignore_missing_imports = True
+follow_imports_for_stubs = True
+
+[mypy-pathway.*]
+check_untyped_defs = True
```

Comment:
I am also not sure if we want to check pathway with a stricter setting than on the default level.

Reply:
done

Reply:
We want to have `check_untyped_defs = True` globally.